### PR TITLE
B1 home screen

### PR DESCRIPTION
**Pull request by NaimGanem**

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
-import {BaseView} from './src/uiKit/BaseView';
-import TimerScreen from './src/screens/Timer/TimerScreen';
-
-function App(): React.JSX.Element {
+const App = () => {
   return (
-    <BaseView>
-      <TimerScreen />
-    </BaseView>
+    <SafeAreaView style={styles.container}>
```
Comment (henaharon) [inline App.tsx:8]: BaseView is already implementing <SafeAreaView 
pls use BaseView
Screenshot 2024-06-21 at 9 30 41

File: src/assets/icons/backgroundVideoCover3@3x.png
Comment (henaharon) [inline src/assets/icons/backgroundVideoCover3@3x.png:1]: Why have you uploaded twice? icons and images?

File: src/assets/images/backgorund3.png
Comment (henaharon) [inline src/assets/images/backgorund3.png:1]: Why have you uploaded twice? icons and images?

File: src/screens/homepage/HomePage.js [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+            </View>
+            <View style={styles.box}>
+                <Image source={first} style={styles.Listlogo} />
+                <Text style={styles.name}>צאט עם נציג</Text>
+            </View>
+        </ScrollView>
+    );
+};
+
+const SecondListComponent = () => {
```
Comment (henaharon) [inline src/screens/homepage/HomePage.js]: Change SecondListComponent to meaningful name

File: src/screens/homepage/HomePage.js [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+const accedent = require('../../assets/images/accedent.png');
+const DashedLine = () => {
+    return (
+        <View style={styles.dashedLineContainer}>
+            <View style={styles.dashedLine} />
+        </View>
+    );
+};
+
+const ListComponent = () => {
```
Comment (henaharon) [inline src/screens/homepage/HomePage.js]: Change ListComponent to meaningful name

File: src/screens/homepage/HomePage.js [hunk 4]
```diff
@@ -0,0 +1,326 @@
...
+        </ScrollView>
+    );
+};
+
+const HomePage = () => {
+    return (
+        <ScrollView contentContainerStyle={styles.scrollViewContent}>
+                <ImageBackground source={backgroundShape} style={styles.backgroundImage}>
+                    <View style={styles.sideMenuButton}>
+                        <Image source={require('../../assets/images/menu.png')} style={{ width: 24, height: 24 }} />
```
Comment (henaharon) [inline src/screens/homepage/HomePage.js]: style={{ width: 24, height: 24 }} 
 No inline style
 
 {require('../../assets/images/menu.png')}
 add this import out side like in lines 3-17

File: src/screens/homepage/HomePage.js [hunk 5]
```diff
@@ -0,0 +1,326 @@
...
+                    </View>
+                    <View style={styles.wrapper}>
+                        <View style={styles.textContainer}>
+                            <Text style={styles.text1}>קבלת שירות</Text>
```
Comment (henaharon) [inline src/screens/homepage/HomePage.js]: Export all texts to text object

'קבלת שירות'

File: src/screens/homepage/HomePage.js [hunk 6]
```diff
@@ -0,0 +1,326 @@
...
+    },
+    SbottomText: {
+        fontSize: 18,
+        fontWeight: 'bold',
+        color: 'black',
+        alignSelf: 'flex-end',
+        marginTop: -40,   
+    },
+    sun: {
+        width: 245,
```
Comment (henaharon) [inline src/screens/homepage/HomePage.js:282]: use responsive sizes
not fixed

File: src/screens/homepage/HomePage.js [hunk 7]
```diff
@@ -0,0 +1,326 @@
...
+                            <Image source={accedent} style={styles.Image} />
+                        </View>
+                        <Image source={sunImage} style={styles.sun} />
+                    </View>
+                </ImageBackground>
+        </ScrollView>
+    );
+};
+
+const styles = StyleSheet.create({
```
Comment (henaharon) [inline src/screens/homepage/HomePage.js:126]: Edit all styles to not use fixed sizes

**Comment by NaimGanem** [comment]
https://drive.google.com/file/d/1hE3lIGzOs_PcLEbEfv-Er_R14nFw-q-3/view?usp=drive_link

**Comment by NaimGanem** [comment]
https://drive.google.com/file/d/1hE3lIGzOs_PcLEbEfv-Er_R14nFw-q-3/view?usp=sharing

**Review by henaharon — APPROVED** [review]

**Pull request merged**